website/content/docs/configuration/storage/raft.mdx

```diff
@@ -51,20 +51,20 @@ set [`disable_mlock`](/vault/docs/configuration#disable_mlock) to `true`, and to
 - `node_id` `(string: "")` - The identifier for the node in the Raft cluster.
   This value can be overridden by setting the `VAULT_RAFT_NODE_ID` environment variable.
 
-- `performance_multiplier` `(integer: 0)` - An integer multiplier used by
+- `performance_multiplier` `(integer: 0)` - An integer multiplier used by 
   servers to scale key Raft timing parameters. Tuning this affects the time it
```

Comment:
```suggestion
  servers to scale key Raft timing parameters, where each increment translates to approximately 1 – 2 seconds of delay. For example, setting the multiplier to "3" translates to 3 – 6 seconds of total delay.  Tuning  the multiplier affects the time it
```

The specific numbers outlined in @banks 's comment are super helpful. I think it's worth adding a note with a specific example here so you don't have to explain it again later. Also updated the language for accessibility (active voice)